<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SemanticUI</class>
 <widget class="QWidget" name="SemanticUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextBrowser" name="QuatText">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>375</width>
     <height>550</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>7</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SimSun-ExtB</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>生成的四元式</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>10</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SimSun-ExtB</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>变量表</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="VarText">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>50</y>
     <width>375</width>
     <height>225</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="FuncText">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>370</y>
     <width>375</width>
     <height>225</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>310</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SimSun-ExtB</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>函数表</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
